qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -640,6 +640,30 @@ def display_template(self, study, user, msg, msg_level, full_access,
                     sub_tab=sub_tab,
                     prep_tab=prep_tab)
 
+    def delete_study(self, study, user, callback):
+        """Delete study
+
+        Parameters
+        ----------
+        study : Study
+            The current study object
+        user : User
+            The current user object
+        callback : function
+            The callback function to call with the results once the processing
+            is done and it fails
+        """
+        study_id = study.id
+
+        try:
+            Study.delete(study_id)
+            self.redirect('/study/list/')
```

Comment:
Can we redirect and show a message saying that the study has been successfully removed? There is no feedback if it actually was successfully removed or not.